__tests__/database/schema/ChannelListSchema.test.ts

```diff
@@ -552,12 +553,13 @@ describe('TESTING ChannelListSchema', () => {
         rawJson: expect.any(Object),
         user: null,
         members: null,
-        channelPicture: '',
+        channelPicture:
+          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
         expiredAt: null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `channelPicture` now has a default profile picture URL instead of being an empty string. This is a good change as it provides a fallback image for channels without a specific picture. However, ensure that the provided URL is reliable and won't cause any issues if it becomes unavailable.
```diff
-        channelPicture: 'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+        channelPicture: 'reliable_default_profile_picture_url',
```
Please replace `'reliable_default_profile_picture_url'` with an actual URL that you trust will always be available.


<!-- This is an auto-generated comment by OSS CodeRabbit -->